tilings/strategy_pack.py

```diff
@@ -38,6 +41,47 @@ def replace_list(strats):
             iterative=self.iterative,
         )
 
+    def setup_subclass_verification(self, start_tiling: "Tiling") -> "TileScopePack":
+        """
+        If the subclass verification strategy already has a list of perms to check,
+        we leave it alone. Otherwise we:
+         - compute the cell basis for each cell
+         - consider the permutations avoiding some cell basis
+         - add to the list of perms to check any perm that avoids some cell basis and
+           has length strictly smaller than the maximum length cell basis element.
+        """
+
+        def alter_list(strats) -> None:
+            """
+            Find subclass verification and alter its perms_to_check variable.
+            """
+            for strategy in strats:
+                if isinstance(strategy, strat.SubclassVerificationStrategy):
+                    if strategy.perms_to_check is None:
```

Comment:
We should be careful to create a new strategy and replacing the other instead of editing this strategy in place. Editing in place will create problem if we use the same pack in multiple searcher at the same time